migrations/Version20250102154204.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250102154204 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'rollback fake signalement date_entree if another real date has been entered';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('UPDATE `signalement` SET `date_entree` = \'2012-03-06\' WHERE id = 49266');
```

Comment:
Est-ce qu'il ne faut pas ajouter la ligne

        `$this->skipIf(true, 'No need to execute this migration in localhost or review app');`

comme dans `migrations/Version20241023122343.php` ?

Reply:
On pourra le mettre quand elle sera déjà passée, non ? parce que si on met skipif à true, ça passera jamais, non ?

Reply:
il n'y a pas un moyen de vérifier le domaine ?

Reply:
genre
```
        $this->skipIf(
            getenv('APP_ENV') !== 'prod',
            'Cette migration ne s’exécute qu’en environnement de production.'
        );
```

ou 

```
    $databaseName = $this->connection->getDatabase();
    $this->skipIf(
        $databaseName !== 'nom_base_production',
        'Cette migration ne s’exécute que sur la base de données de production.'
    );
```

Reply:
Ok j'ai ajouté le test sur l'environnement

Reply:
ok top